pkg/verify/verify.go

```diff
@@ -74,7 +72,7 @@ var Reject Interface = rejectVerifier{}
 // if each provided public key has signed the release image digest. The signature may be in any
 // store and the lookup order is internally defined.
 //
-func LoadFromPayload(update *payload.Update) (Interface, error) {
+func LoadFromPayload(update *payload.Update, transport http.RoundTripper) (Interface, error) {
```

Comment:
i think passing the entire transport is not get an API for this function.

maybe we can think about passing a little more explicit item? transport.Config maybe, or the tlsConfig

Reply:
I would figure you would need to pass both the `tlsConfig` and `proxyURL`.
Something like what is in: 
https://github.com/jcpowermac/cluster-version-operator/blob/master/pkg/cvo/availableupdates.go

Then create the http client:
https://github.com/jcpowermac/cluster-version-operator/blob/d4e138fac270b557fb9d2592d000de40fcd9affd/pkg/cincinnati/cincinnati.go#L64-L72